src/autorest-core/test/blaming.ts

```diff
@@ -13,30 +13,32 @@ import { parse } from "../lib/ref/jsonpath";
 
 @suite class Blaming {
 
-  @test @skip @timeout(0) async "end to end blaming with literate swagger"() {
+  @test @timeout(0) async "end to end blaming with literate swagger"() {
     const autoRest = new AutoRest(new RealFileSystem(), ResolveUri(CreateFolderUri(__dirname), "../../test/resources/literate-example/readme-composite.md"));
 
     autoRest.AddConfiguration({ "use-extension": { "@microsoft.azure/autorest-classic-generators": `${__dirname}/../../../core/AutoRest` } })
     // PumpMessagesToConsole(autoRest);
     const view = await autoRest.view;
     assert.equal(await autoRest.Process().finish, true);
 
+    const keys = Object.keys((view.DataStore as any).store);
```

Comment:
oh no, just don't wanna spam PRs... just taking the chance to reenable some tests that were broken over the past weeks due to protocol and format changes.